misago/legal/models.py

```diff
@@ -1 +1,127 @@
 # Empty models file that triggers migrations for this app
+from django.db import models
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+import datetime
+
+
+class NoActiveLegalAgreement(ValidationError):
+    pass
+
+
+class BaseModel(models.Model):
+    created = models.DateTimeField(auto_now_add=True, editable=False)
+    modified = models.DateTimeField(auto_now=True, editable=False)
+
+    class Meta:
+        abstract = True
+
+
+class LegalAgreementManager(models.Manager):
+    def get_current(self):
+        try:
+            return self.get(active=True)
+        except self.model.DoesNotExist:
+            raise NoActiveLegalAgreement(
+                u'Please create an active LegalAgreement'
+            )
+
+
+class LegalAgreement(BaseModel):
+    IMP_HIGH = 2
+    IMP_MEDIUM = 1
+    IMP_LOW = 0
+    IMPORTANCE = (
+        (IMP_HIGH, 'High'),
+        (IMP_MEDIUM, 'Medium'),
+        (IMP_LOW, 'Low'),
+    )
+    TOS = 'terms_of_service'
+    PRIVACY = 'privacy_policy'
+    Legal = (
+        (TOS, 'Terms of service'),
+        (PRIVACY, 'Privacy Policy'),
+    )
+    active = models.BooleanField(
```

Comment:
Please separate constants from fields with empty line. Also boolean flags should have `is_` prefix, eg `is_active`.